e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -481,6 +488,79 @@ describeIfIAMV2p1('projects API: applying project', () => {
       expect(response.body).to.have.length(2);
     });
   });
+
+  const attributeValuesMap = new Map([
```

Comment:
Please add this comment below.
(Note this identical comment is used elsewhere, so I think it is a good idea to reuse it verbatim. If you want to revise it, then suggest revising it in both places.)
```
must correspond to enum type in automate-gateway/.../common/rules.proto
```